index.ts

```diff
@@ -419,32 +470,45 @@ module.exports.pixelToLatitude = pixelToLatitude
  * @example
  * var ll = lonlat.fromPixel({x: 40000, y: 50000}, 9) //= {lon: -70.13671875, lat: 39.1982053488948}
  */
-module.exports.fromPixel = function fromPixel (pixel, zoom) {
+export function fromPixel(pixel: Point, zoom: number): LonLatOutput {
   return {
-    lon: pixelToLongitude(pixel.x, zoom),
-    lat: pixelToLatitude(pixel.y, zoom)
+    lat: pixelToLatitude(pixel.y, zoom),
+    lon: pixelToLongitude(pixel.x, zoom)
   }
 }
 
-function floatize (lonlat) {
-  var lon = parseFloatWithAlternates([lonlat.lon, lonlat.lng, lonlat.longitude])
-  var lat = parseFloatWithAlternates([lonlat.lat, lonlat.latitude])
+// Can the various ways in which lat/long pairs can be expressed to this
+// method be expressed as a type?
+// eslint-disable-next-line complexity
+function floatize(lonlat: Record<string, unknown>): LonLatOutput {
+  const lon = parseFloatWithAlternates([
+    lonlat.lon,
+    lonlat.lng,
+    lonlat.longitude
+  ])
+  const lat = parseFloatWithAlternates([lonlat.lat, lonlat.latitude])
   if ((!lon || lon > 180 || lon < -180) && lon !== 0) {
-    throw new Error('Invalid longitude value: ' + (lonlat.lon || lonlat.lng || lonlat.longitude))
+    throw new Error(
+      'Invalid longitude value: ' +
+        (lonlat.lon || lonlat.lng || lonlat.longitude)
+    )
   }
   if ((!lat || lat > 90 || lat < -90) && lat !== 0) {
-    throw new Error('Invalid latitude value: ' + (lonlat.lat || lonlat.latitude))
+    throw new Error(
+      'Invalid latitude value: ' + (lonlat.lat || lonlat.latitude)
+    )
   }
-  return {lon: lon, lat: lat}
+  return { lat: lat, lon: lon }
```

Comment:
Nit: is there a reason this line doesn't use the simpler syntax?
```suggestion
  return { lat, lon }
```

Reply:
Good catch thank you! I think this may be a relic from before es6.